ci/clean-old-temporary-docker-images.py

```diff
@@ -0,0 +1,31 @@
+import os
+import requests
+from datetime import datetime, timedelta
+
+DOCKERHUB_TOKEN = os.environ["DOCKERHUB_TOKEN"]
+DELETE_AFTER_DAYS = os.environ["DELETE_AFTER_DAYS"]
+
+def get_repo_tags(token):
+    url = f"https://hub.docker.com/v2/repositories/scylladb/gocql-extended-ci/tags/"
+    headers = {"Authorization": f"Bearer {token}"}
+    response = requests.get(url, headers=headers)
+    return response.json()["results"]
```

Comment:
Please hande bad response gracefully
```suggestion
        if response.status_code != 200:
        print(f"Failed to get tags, Status Code: {response.status_code}, {response.text}")
        return []
    return response.json()["results"]
```